buildspec.yaml

```diff
@@ -18,6 +18,9 @@ phases:
       - /bin/bash ./fetch-dotnet-snk.sh
   build:
     commands:
+      # we bump here so that our master version won't be identical to the latest published version during tests.
+      # otherwise this causes problems with verdaccio mirroring.
+      - '[ ${GIT_BRANCH} = "master" ] && /bin/bash ./bump-candidate.sh'
```

Comment:
I feel this might become a problem if we ever change the name of our primary branch to `main`. 

Can we externalize this in some way? Add Yet Another variable? `REGRESSION_TEST_BRANCH` or something?